parquet-hadoop/src/main/java/org/apache/parquet/format/converter/ParquetMetadataConverter.java

```diff
@@ -480,6 +480,10 @@ private void addRowGroup(ParquetMetadata parquetMetadata, List<RowGroup> rowGrou
           columnMetaData.getTotalSize(),
           columnMetaData.getFirstDataPageOffset());
       columnChunk.meta_data.dictionary_page_offset = columnMetaData.getDictionaryPageOffset();
```

Comment:
Use setDictionary_page_offset instead of `columnChunk.meta_data.dictionary_page_offset =`. That will automatically invoke setDictionary_page_offsetIsSet and we don't need to call it explicitly.

Reply:
Done